test/utils.test.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line spaced-comment
+/* eslint-env node, jest */
+/* global BigInt */
```

Comment:
Is there something we could do to avoid having to do this? Upgrading ESLint / changing ESLint configs?